charts/kubernetes/templates/topology.yaml

```diff
@@ -120,7 +125,14 @@ spec:
                   'logs': [{'name': 'Kubernetes', 'type': 'KubernetesPod'}],
                   'status': k8s.getHealth(r).status == "Healthy" ? "healthy" : "unhealthy",
                   'status_reason': k8s.getHealth(r).message,
-                  'properties': k8s.podProperties(r),
+                  'properties': [
+                    {"name": "image", "text": r.spec.containers[0].image},
+                    {"name": "cpu", "max": k8s.podMaxCPUMillicores(r), "unit": "millicores", "headline": true},
```

Comment:
Were we not going todo this as `k8s.getResourcesLimit|Request('cpu')`

Reply:
Updated, sorry that discussion slipped out of my mind